payu/experiment.py

```diff
@@ -638,7 +638,13 @@ def archive(self):
                 if os.path.isdir(res_path):
                     shutil.rmtree(res_path)
 
-        if self.config.get('collate', True):
+        do_collate = self.config.get('collate',{})
```

Comment:
I think the place to do the first part of this (convert bool to dict) might be in the `read_config` function of `payu.fsops.py`.